test/artefacts/landlord-immigration-check/B1 1PW/yes/yes/non-eea/no/yes.txt

```diff
@@ -1,10 +1,10 @@
 
 
-This person can rent the property.
+This person can currently rent the property.
 
-You need to make a copy of the [relevant parts of the documents](/check-tenant-right-to-rent-documents) and keep the copies for 12 months after the tenancy has ended.
+You need to make a copy of the [relevant parts of the documents](/check-tenant-right-to-rent-documents).
 
-You don’t have to make this check again.
+If the tenant's right to stay in the UK is for less than 12 months, check again just before 12 months' time. Otherwise check again just before their right to stay in the UK runs out.
```

Comment:
@ikennaokpala I have left a comment for Polly on the Trello card to double check this.